src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -964,88 +964,40 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: SlickDataSo
-            val errors = for (entityValue <- entityInputs.inputResolutions if entityValue.error.isDefined) yield (AttributeString(entityValue.error.get))
-            WorkflowFailure(entityInputs.entityName, header.entityType, entityInputs.inputResolutions, errors)
-          } ++ failedWorkflowSubmissions
-
-          val submission = Submission(submissionId = submissionId,
-            submissionDate = DateTime.now(),
-            submitter = RawlsUser(userInfo),
-            methodConfigurationNamespace = submissionRequest.methodConfigurationNamespace,
-            methodConfigurationName = submissionRequest.methodConfigurationName,
-            submissionEntity = Option(AttributeEntityReference(entityType = submissionRequest.entityType, entityName = submissionRequest.entityName)),
-            workflows = succeededWorkflowSubmissions,
-            notstarted = failedWorkflows,
-            status = if (succeededWorkflowSubmissions.isEmpty) SubmissionStatuses.Done else SubmissionStatuses.Submitted
-          )
-
-          dataAccess.submissionQuery.create(workspaceContext, submission) map { _ =>
-            val workflowReports = succeededWorkflowSubmissions.map { workflow =>
-              WorkflowReport(workflow.workflowId, workflow.status, workflow.statusLastChangedDate, workflow.workflowEntity.map(_.entityName).getOrElse("*deleted*"), workflow.inputResolutions)
-            }
+        val workflows = successes map { entityInputs =>
+          Workflow(workflowId = None,
+            status = WorkflowStatuses.Queued,
+            statusLastChangedDate = DateTime.now,
+            workflowEntity = Option(AttributeEntityReference(entityType = header.entityType, entityName = entityInputs.entityName)),
+            inputResolutions = entityInputs.inputResolutions)
+        }
+        val workflowFailures = failures.map { entityInputs =>
+          val errors = for (entityValue <- entityInputs.inputResolutions if entityValue.error.isDefined) yield (AttributeString(entityValue.error.get))
+          WorkflowFailure(entityInputs.entityName, header.entityType, entityInputs.inputResolutions, errors)
+        }
 
-            RequestComplete(StatusCodes.Created, SubmissionReport(submissionRequest, submission.submissionId, submission.submissionDate, userInfo.userEmail, submission.status, header, workflowReports, failures))
-          }
+        val submission = Submission(submissionId = submissionId,
+          submissionDate = DateTime.now(),
+          submitter = RawlsUser(userInfo),
+          methodConfigurationNamespace = submissionRequest.methodConfigurationNamespace,
+          methodConfigurationName = submissionRequest.methodConfigurationName,
+          submissionEntity = Option(AttributeEntityReference(entityType = submissionRequest.entityType, entityName = submissionRequest.entityName)),
+          workflows = workflows,
+          notstarted = workflowFailures,
+          status = SubmissionStatuses.Submitted
```

Comment:
We no longer do anything special for the all-failures case.  Looks like this was intentional, just wanted to make sure.